tests/isomorphism_problem.py

```diff
@@ -13,134 +13,222 @@
+                        write_csv_line(
+                            csv_filepath, [file, str(i), str(j), True,
+                                           "{0:.3f}".format(start_isomorph - end_isomorph),
+                                           "{0:.3f}".format(start_amount_isomorphisms - end_amount_isomorphisms),
+                                           "{0}".format(remark)]
+                        )
+                        if show_passed_results:
+                            passed(remark)
                 else:
-                    if show_passed_results:
-                        passed("Graphs are not isomorph")
-
-            print("Processing time is_isomorph(G, H): " + str(round((end_isomorph - start_isomorph) * 1000, 3)) + " ms")
-            print("Processing time amount_of_isomorphisms(G, H): " + str(round((end_amount_isomorphisms - start_amount_isomorphisms) * 1000, 3)) + " ms")
-            print('')
-
-print('---------------------------')
-print("Statistics of test:")
-print('---------------------------')
-print("Amount of graphs tested: " + str(graph_count))
-print("Amount of graphs not colored correctly: " + str(error_count))
-print('')
-if error_count > 0:
-    fail('TEST FAILED')
-else:
-    passed('TEST PASSED')
-print('')
+                    if boolean_isomorph:
+                        remark = "[FAIL] Graphs are not isomorph, is_isomorph determined they were"
+                        fail(remark)
+                        write_csv_line(
+                            csv_filepath, [file, str(i), str(j), False,
+                                           "{0:.3f}".format(start_isomorph - end_isomorph), "{0:.3f}".format(start_amount_isomorphisms - end_amount_isomorphisms),
+                                           "{0}".format(remark)]
+                        )
+                        error_count += 1
+                    else:
+                        remark = "[PASS] Graphs are not isomorph"
+                        write_csv_line(
+                            csv_filepath, [file, str(i), str(j), True,
+                                           "{0:.3f}".format(start_isomorph - end_isomorph),
+                                           "{0:.3f}".format(start_amount_isomorphisms - end_amount_isomorphisms),
+                                           "{0}".format(remark)]
+                        )
+                        if show_passed_results:
+                            passed(remark)
+
+                total_time += (end_isomorph - start_isomorph) + (end_amount_isomorphisms - start_amount_isomorphisms)
+                #print("Processing time is_isomorph(G, H): " + str(round((end_isomorph - start_isomorph) * 1000, 3)) + " ms")
+                #print("Processing time amount_of_isomorphisms(G, H): " + str(round((end_amount_isomorphisms - start_amount_isomorphisms) * 1000, 3)) + " ms")
+                #print('')
+
+    print('---------------------------')
```

Comment:
Volgens mij mag dit ook weg, deze informatie zit al in determine_outcome. Als je alle tests runt dan past deze informatie er ook niet echt tussen.

Reply:
ik heb deze stukken gerefactored zodat ze alleen zichtbaar zijn als je de optie write_stdout_passed en/of write_stdout_fail op True zet.